BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift

```diff
@@ -150,23 +172,18 @@ public final class ConvivaAnalytics: NSObject {
      relies on the players source we can't ensure that all metadata attributes are present at session creation.
      Therefore it could be that there will be a 'ContentMetadata created late' issue after conviva validation.
 
-     - Parameters:
-        - assetName: Will be used as contentMetadata.assetName if no source was loaded before. This overrides the
-                     source assetName. If no source was loaded and no assetName is present this method will throw an
-                     error.
+     If no source was loaded and no assetName was set via updateContentMetadata this method will throw an error.
```

Comment:
There could be a source loaded without title, basically.

Reply:
Uh thanks for that. Good catch 🤔 I guess that can happen on all platforms? So this need to be reworked.

Reply:
Actually it's handled. Just the wording is misleading.